common/common.h

```diff
@@ -44,7 +44,9 @@ struct llama_control_vector_load_info;
 //
 // CPU utils
 //
-
+#ifdef _WIN32
+int32_t cpu_get_num_cores_win(bool print_physical_core_num);
+#endif
```

Comment:
Instead of declaring this new function, just inline the windows code in the existing `cpu_get_num_physical_cores`

Reply:
Yes, I have deleted the new function in my latest commit.